@types/modules/pm/groups.d.ts

```diff
@@ -0,0 +1,20 @@
+declare interface PMGroupUpdateRequest {
+    prizeCategoryId: string;
+    prizeCatagoryName?: string;
+    prizeCategoryDescription?: string;
+    prizeCategoryOrder?: Array<string>;
+  }
+
+  declare interface PMGroupCreateRequest extends PMGroupUpdateRequest {
+    prizeCatagoryName: string;
+    prizeCategoryDescription: string;
+    prizeCategoryOrder: Array<string>;
+  }
+
+  declare interface PMGroup extends PMGroupCreateRequest {
+    prizeCategories: { [key: string]: string };
+  }
+
+//   declare interface PMGroupList {
+//     prizeCategories: STPMCompany[];
+//   }
```

Comment:
We have a list for PM Groups right? I think that we'd need this (or I could just be going crazy and forgetting what endpoints we have...).
```suggestion
declare interface PMGroupList {
    groups: PMGroup[];
}
```